src/internal/modules/licences/routes/bills-tab.js

```diff
@@ -4,7 +4,7 @@ const Joi = require('@hapi/joi');
 const controller = require('../controllers/bills-tab');
 const { scope } = require('../../../lib/constants');
 
-const allowedScopes = [scope.billing];
+const allowedScopes = [scope.charging];
```

Comment:
I seem to have both a "charging" and "billing" role in the IDM.

I think we need to ensure all services are using the same one and delete the other - although that's not necessarily within the scope of this ticket.